apps/browser/components/Chat.tsx

```diff
@@ -136,7 +136,7 @@ const Chat: React.FC<ChatProps> = ({
               <ChatInputButton
                 running={isRunning}
                 message={message}
-                handleSend={async () => await handleGoalSubmit(message)}
+                handleSend={() => handleGoalSubmit(message)}
```

Comment:
There's actually a mistake here, `handleGoalSubmit` needs to be async & awaited because `handleStart` needs to be async and awaited as well. Inside of `useEvoService` we need to fix the return typing for `handleStart` to return `Promise<void>` since it `await createChatIdIfNeccessary(chatId);`

Reply:
Fixed and awaiting in appropriate places (not here cause there's no need)